library/include/borealis/swkbd.hpp

```diff
@@ -22,14 +22,48 @@
 #include <functional>
 #include <string>
 
+#ifdef __SWITCH__
+#include <switch.h>
+#endif
+
 namespace brls
 {
 
+enum keyboard_disable_bitmask
+{
+#ifdef __SWITCH__
+    KEYBOARD_DISABLE_NONE         = 0,
+    KEYBOARD_DISABLE_SPACE        = SwkbdKeyDisableBitmask_Space,        ///< Disable space-bar.
```

Comment:
No need for that funky `////<` syntax here

Reply:
yes sorry I missed this, 
I copy/pasted the text from the libnx https://github.com/switchbrew/libnx/blob/3d4f61735ad8515dfb3e9dc0c0c7568babbdec33/nx/include/switch/applets/swkbd.h#L35
it might be use by an automated tool.